Compile errors on mac in PS V3

```
[build] In file included from /Users/flow/workspace/github/theflash/storage/ch/dbms/src/Storages/Page/V3/spacemap/SpaceMap.cpp:19:
[build] /Users/flow/workspace/github/theflash/storage/ch/dbms/src/Storages/Page/V3/spacemap/SpaceMapSTDMap.h:43:25: error: loop variable '[offset, length]' of type 'const std::__1::pair<const unsigned long long, unsigned long long>' creates a copy from type 'const std::__1::pair<const unsigned long long, unsigned long long>' [-Werror,-Wrange-loop-analysis]
[build]         for (const auto [offset, length] : free_map)
[build]                         ^
[build] /Users/flow/workspace/github/theflash/storage/ch/dbms/src/Storages/Page/V3/spacemap/SpaceMapSTDMap.h:43:14: note: use reference type 'const std::__1::pair<const unsigned long long, unsigned long long> &' to prevent copying
[build]         for (const auto [offset, length] : free_map)
[build]              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[build]                         &
[build] 1 error generated.
[build] /Users/flow/workspace/github/theflash/storage/ch/dbms/src/Storages/Page/V3/PageDirectory.cpp:29:10: fatal error: 'openssl/base64.h' file not found
[build] #include <openssl/base64.h>
[build]          ^~~~~~~~~~~~~~~~~~
[build] 1 error generated.
```

--- a/dbms/src/Storages/Page/V3/PageDirectory.cpp
+++ b/dbms/src/Storages/Page/V3/PageDirectory.cpp
@@ -26,7 +26,6 @@
 #include <Storages/Page/V3/WALStore.h>
 #include <Storages/Page/WriteBatch.h>
 #include <common/logger_useful.h>
-#include <openssl/base64.h>
 
 #include <memory>
 #include <mutex>

--- a/dbms/src/Storages/Page/V3/spacemap/SpaceMapSTDMap.h
+++ b/dbms/src/Storages/Page/V3/spacemap/SpaceMapSTDMap.h
@@ -40,7 +40,7 @@ public:
     bool check(std::function<bool(size_t idx, UInt64 start, UInt64 end)> checker, size_t size) override
     {
         size_t idx = 0;
-        for (const auto [offset, length] : free_map)
+        for (const auto & [offset, length] : free_map)
         {
             if (!checker(idx, offset, offset + length))
                 return false;

